test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java

```diff
@@ -21,210 +21,110 @@
  * questions.
  */
 
-/**
+import java.awt.Frame;
+import java.awt.print.PageFormat;
+import java.awt.print.PrinterJob;
+import java.lang.reflect.InvocationTargetException;
+
+import javax.print.PrintService;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.SwingUtilities;
+
+/*
  * @test
  * @bug 4937672 5100706 6252456
  * @key printer
- * @run main/othervm/manual -Djava.security.manager=allow SecurityDialogTest
+ * @summary tests native and cross-platform page and print dialog
+ *          when security manager is installed
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual/othervm -Djava.security.manager=allow SecurityDialogTest
  */
+public class SecurityDialogTest extends Frame {
```

Comment:
```suggestion
public class SecurityDialogTest {
```
It does not need to extend `Frame`.